README.md

```diff
@@ -14,33 +14,32 @@
 </a>
 </p>
 
-Lightweight, flexible server framework written in Swift.
-
-Hummingbird consists of three main components, the core HTTP server, a minimal web application framework and the extension modules.
+Lightweight, flexible, modern server framework written in Swift.
 
 ## HummingbirdCore
 
-HummingbirdCore contains a Swift NIO based HTTP server. You will find the code for it in the [hummingbird-core](https://github.com/hummingbird-project/hummingbird-core) repository. The HTTP server is initialized with a object conforming to protocol `HBHTTPResponder` which defines how your server responds to an HTTP request. The HTTP server can be extended to support TLS and HTTP2 via the `HummingbirdTLS` and `HummingbirdHTTP2` libraries also available in the hummingbird-core repository.
+HummingbirdCore contains a Swift NIO based server framework. The server framework `HBServer` can be used to support many protocols but is primarily designed to support HTTP. By default it is setup to be an HTTP/1.1 server, but it can support TLS and HTTP2 via the `HummingbirdTLS` and `HummingbirdHTTP2` libraries.
```

Comment:
> ..., but it can support TLS and HTTP2 via the `HummingbirdTLS` and `HummingbirdHTTP2` libraries.
Wouldn't `modules` be clearer? As these features are in the hummingbird package itself, this might lead people to search for packages with those names